public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -102,9 +130,20 @@ export class PanelHeader extends Component<Props, State> {
       'grid-drag-handle': !isFullscreen,
     });
 
+    // dedupe on severity
+    const notices: Record<string, QueryResultMetaNotice> = {};
+
+    for (const series of data.series) {
+      if (series.meta && series.meta.notices) {
+        for (const notice of series.meta.notices) {
+          notices[notice.severity] = notice;
+        }
+      }
+    }
```

Comment:
notices with same severity will overwrite each other (otherwise would be flooded with info icons)

Reply:
seems fine for now -- the only better option I could see is switching to a number indicator with the count